analyzer/consensus/queries.go

```diff
@@ -254,16 +254,12 @@ func (qf QueryFactory) DeleteDebondingDelegationsQuery() string {
 	// Network upgrades delays debonding by 1 epoch
 	return fmt.Sprintf(`
 		DELETE FROM %[1]s.debonding_delegations
-			WHERE (ctid) IN (
-				SELECT ctid
+			WHERE id = (
```

Comment:
Why was it important to switch from `ctid` to a newly-introduced `id`?

Reply:
Supposedly, `ctid` is not a stable identifier at least [long term](https://www.postgresql.org/docs/15/ddl-system-columns.html). There was a bit of future/bullet proofing behind this decision. The main source of contention was the race condition between different goroutines (routine A inserting and updating epochs, and routine B here querying based on epoch state). This PR addressed the main problem by sourcing epoch information from the block as opposed to the current DB state.